clients/fides-js/src/fides.ts

```diff
@@ -14,22 +14,27 @@
  * </script>
  * ```
  */
-import { gtm } from "./integrations/gtm";
-import { meta } from "./integrations/meta";
-import { shopify } from "./integrations/shopify";
-import { ConsentConfig } from "./lib/consent-config";
-import { getConsentContext } from "./lib/consent-context";
-import { CookieKeyConsent, getConsentCookie, makeDefaults } from "./lib/cookie";
+import {gtm} from "./integrations/gtm";
+import {meta} from "./integrations/meta";
+import {shopify} from "./integrations/shopify";
+import {ConsentConfig} from "./lib/consent-config";
+import {getConsentContext} from "./lib/consent-context";
+import {InitFidesConsent} from "./lib/consent";
+import {CookieKeyConsent, getConsentCookie, makeDefaults} from "./lib/cookie";
+import {ConsentBannerOptions} from "~/lib/consent-types";
+import {getBannerOptions} from "~/lib/consent-utils";
 
 /**
  * Configuration options for the fides.js script
  */
 export interface FidesConfig {
   consent: ConsentConfig;
+  bannerOptions?: ConsentBannerOptions;
 }
 
 type Fides = {
   consent: CookieKeyConsent;
+  getBannerOptions: ConsentBannerOptions;
```

Comment:
Shouldn't this be a function, like `getBannerOptions: () => ConsentBannerOptions`